### ISSUE TITLE
Mismatch in winetricks version

### ISSUE THREAD

**Issue by bearcatjamboree** [Winetricks/winetricks#1826]
When I installed winetricks I get the following:

```
% brew install winetricks
==> Downloading https://ghcr.io/v2/homebrew/core/winetricks/manifests/20210825
Already downloaded: /Users/xxxxx/Library/Caches/Homebrew/downloads/ba4a68a8f7f3da22f6a0178a4ddcc832740a967ab3514620d8095e77d1f34b5b--winetricks-20210825.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/winetricks/blobs/sha256:322c2e6e1dd72073bfb03a230820eb039592c016103d0c442739e9f6c9f3470b
Already downloaded: /Users/xxxxx/Library/Caches/Homebrew/downloads/22c6ee5a03833e651be5d37653a29f619b04f58f7efb25b2c6337c1fcfdd9ce6--winetricks--20210825.all.bottle.tar.gz
==> Pouring winetricks--20210825.all.bottle.tar.gz
🍺  /opt/homebrew/Cellar/winetricks/20210825: 6 files, 905.3KB
```

```
% which winetricks
/opt/homebrew/bin/winetricks
```

```
% winetricks --version
20210206-next - sha256sum: 0769bbcdbd92c407f4eacaa85acc1339f607dbeafe2febd1be0912034c7af3a1
```

```
% winetricks win81
warning: taskset/cpuset not available on your platform!
------------------------------------------------------
warning: Your version of wine  is no longer supported upstream. You should upgrade to 5.x
------------------------------------------------------
```

```
% wine64 --version
wine-6.0.1
```

When I download the script direct from here I get the same behavior.  Did someone post a new release with the same code from 20210206?

**Comment by austin987** [comment]
This should be resolved with the latest release (20220411).

TL;DR: my development environment was messed up for a bit, leading to bad tags/releases.
